docs/source/more.rst

```diff
@@ -103,7 +103,25 @@ TBD
 Creating models instances and classes from dicts
```

Comment:
The goal of this topic is to document the `simple_model.builder` module and its three functions: `model_class_builder`, `model_builder`, and `model_many_builder`. The `Model.as_dict()` method is also part of the "Converting models to dict" because both `to_dict()` function and the method are different ways to achieve the same objective.

I should have described better those topics, sorry for misleading you :see_no_evil:. Would you please move the content of this PR to the "Converting models to dict" section of the docs?

Reply:
Done!